lib/fastlane/plugin/wpmreleasetoolkit/helper/promo_screenshots_helper.rb

```diff
@@ -28,6 +28,8 @@ def initialize
           message << 'Aborting.'
           UI.user_error!(message)
         end
+
+        UI.user_error!('`drawText` not found – install it using `brew install automattic/build-tools/drawText`.') unless system('which drawText')
```

Comment:
`shellcheck` taught me recently that `command -v drawText` was more recommended than `which drawText` 🤷 

See https://github.com/koalaman/shellcheck/wiki/SC2230

Reply:
Ah yes, this is even better!

Addressed in 5e5bf9b636f1b70c877ebc18d6d163772bc89ddd